cpu/stm32f3/ldscripts/stm32f303cc.ld

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32f3
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32F303CC
+ *
+ * @author          Sebastian Meiling <s@mlng.net>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x08000000, LENGTH = 256K
+    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 40K
+    ccmram (rwx): ORIGIN = 0x10000000, LENGTH = 8K
+    cpuid (r)   : ORIGIN = 0x1FFFF7AC, LENGTH = 12
+}
+
+_cpuid_address = ORIGIN(cpuid);
+
+INCLUDE cortexm_base.ld
```

Comment:
Is a custom linker script really needed?

Reply:
uh, don't know - old PR need to look again, any suggestions? Also with BOARDSDIR in #12183 this could remain in a separate repo as this boards does not seem to be widely available (though I bought one a while ago).